### ISSUE TITLE
Spanish translation work for updated booster shots (main vaccines page)

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#46878]
## Issue Description
We made updates to our [main vaccine page](https://www.va.gov/health-care/covid-19-vaccine/) on 9/9/22 about updated booster shots. We need to update our Spanish page with the new translated copy.

Copy of word doc that was sent: [https://app.zenhub.com/files/133843125/d37a1766-c906-4c22-ab23-1fea6e405a8e/download](https://app.zenhub.com/files/133843125/d37a1766-c906-4c22-ab23-1fea6e405a8e/download)

## Tasks
- [x] Send text to translators (Randi)
- [x] Input content on Spanish page (Aliyah)
- [x] Copyedit (Megan)
- [x] Publish (Megan)

## Acceptance Criteria
- [x] Live Spanish page is updated with newest content

**Comment by aliyahblackmore** [comment]
Thanks @RLHecht!

@megzehn this is ready for you!
Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=16359

**Notes/Flags:** 

There were some 711 changes made to this page, as part of the audit. This is reflected in the revision log.

1. In the page intro, “Lea esta página para saber cómo obtener la vacuna COVID-19 en VA” is slightly different than the English in the document that we sent to them. This wasn’t highlighted, but wondering if there’s a need to change this at all. Based on my Spanish knowledge, the latter part of the sentence would remain but we would need an update for “Find out.” Also wondering if this is an instance of this current Spanish text being what would make sense when this is read in Spanish? cc: @RLHecht
2. As I’ve done previously, I’ve included the numeral for numbers instead of the spelled out version (i.e. “3” instead of “tres”). At another point in the translation “2” is used instead of “dos,” so I wanted this to be consistent. CC: Randi, wondering if we should ask the translators if there is a standard for numbers in Spanish that we should follow or if it’s okay to use the numeral instead of having it spelled out. This wasn't consistent on the page.
3. In the info box, the text that came back for the sentence that begins with “If you’re an eligible Veteran or family member…” includes “below,” which is sighted or directional language. I put this is Google translate as well as in[ SpanishDictionary ](https://www.spanishdict.com/)(another Spanish language resource) and both translate this text to “ if you meet the requirements below:” Wondering if we can or should ask if another translation can work to avoid this sighted/directional language. I’ve left this as is for the time being. CC: @RLHecht 
4. Also in the info box, we are missing “These vaccines are effective at preventing severe illness from COVID-19.” This was highlighted in the translation that we sent over to them. I’ve just added in the sentence they sent over for the time being. CC: @RLHecht 
5. When I selected the link for “Learn how to get your primary COVID-19 vaccine”(Aprenda como puede recibir su vacuna de serie primaria contra el COVID-19) on the live English page, it jumped to the H2 “Who can get a COVID-19 vaccine at VA.” I added in the jump link to this section, but I’m wondering if the jump link on the live English page would instead go to “How to get a COVID-19 Vaccine at VA?” I’ve left this as is for now, to match the jump link on the English page, but I wonder if the link on the English page should be pointing to the “How” H2 instead.
6. In the info box, the link “Entérese de cómo puede recibir una prueba de COVID-19 a través del VA” leads to an English page. In line with other portions of the page, I added in “([en inglés](https://www.va.gov/resources/covid-19-testing-at-va/))” after the link.
7. The second bullet point in accordion “Should I get a COVID-19 vaccine if I’ve already had COVID-19?” was removed on the live English page and the opening sentence is now a bit different.  To move this along, I added in the “Yes” (Si) and removed “3” so that the text is most closely aligned with the live English text. This was a smaller text change, but wondering if we should have the translators translate this just to confirm that this is correct. The sentence is "Yes. Data from clinical trials show that the authorized COVID-19 vaccines are safe in people who’ve been infected with the virus that causes COVID-19 in the past." Wondering if this sentence difference is part of more recent changes, as it’s not highlighted? I did a comparison of revisions from earlier in the month and the most recent, and I'm not seeing this as a recent change. CC: @RLHecht 
9. In H3 “ How to get your first or second vaccine dose” the bulleted list in “Here’s what to know before you arrive” - this content that was removed:  “Si no recibe atención en VA, le recomendamos que se registre en línea al menos una hora antes de ir a una clínica sin cita previa. Esto le ahorrará tiempo cuando llegue. Inscríbase para recibir la vacuna COVID-19 en VA (en inglés)” as it is no longer on the live English page. cc: @RLHecht
